### PR TITLE
Feature : Location added.

### PR DESCRIPTION
**Pull request by tusharkanakagiri**

### CODE REVIEW

File: node/app.js [hunk 1]
```diff
@@ -332,15 +332,18 @@ function receivedMessage(event) {
...
+            type : ['hospital']
         };
 
         nearBySearch(parameters, function (error, response) {
             if(error) {
               console.log(error);
               return false;
             }
-            console.log(response);
+            console.log(response['results'][1]['name']);
```
Comment (Sparshith) [inline node/app.js]: Sir, remove console logs.

File: node/app.js [hunk 2]
```diff
@@ -397,7 +400,7 @@ function receivedPostback(event) {
 
   console.log(payload);
   if(payload == 'get_started') {
-    sendTextMessage(senderID, "Hi, Can you please share your location?");
+    sendQuickReply(senderID, 1);
```
Comment (Sparshith) [inline node/app.js]: Replace use case numbers with meaningful strings.

File: node/app.js [hunk 3]
```diff
@@ -733,30 +736,53 @@ function sendQuickReply(recipientId) {
 
   }, function (error, response, body) {
     if (!error && response.statusCode == 200) {
+
+      var quickReplyMessage = '';
+
+        switch(use_case) 
+      {
```
Comment (Sparshith) [inline node/app.js]: Indentation.

File: node/config/default.json [hunk 4]
```diff
@@ -1,6 +1,8 @@
 {
-    "appSecret": "",
-    "pageAccessToken": "",
-    "validationToken": "",
-    "serverURL": ""
+    "appSecret": "10d0355e8e607a8c5185d713b1686c1f",
```
Comment (Sparshith) [inline node/config/default.json:2]: This is not supposed to show up. This should be in your gitignore.

Comment (Sparshith) [reply node/config/default.json:2]: Use this command to add an existing file to your gitignore without deleting it from the remote repo. 

git update-index --assume-unchanged FILENAME

**Comment by tusharkanakagiri** [comment]
PR Changes done.

**Comment by Sparshith** [comment]
Congratulations sir 💯

**Pull request merged**